eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -20,7 +20,7 @@
     "NodeTestVersion": [
       "18.x",
       "20.x",
-      "21.x"
+      "21.2.0"
```

Comment:
Why pin to `21.2.0` rather than `21.3.0`?

Reply:
21.3.0 has another regression that broke some identity tests: https://github.com/nodejs/node/issues/50989